File: Front/src/app/models/Hotel.Model.ts
export class HotelModel {
    hotel_id: number;
    hotel_name: string;
    max_reservations: number;
    phone_number: string;
    email: string;
    capacity: number;
    img_url: string;
    price: number;

    constructor()
    {
        this.hotel_id = 0;
        this.hotel_name = '';
        this.max_reservations = 0;
        this.phone_number = '';
        this.email = '';
        this.capacity = 0
        this.img_url = '';
        this.price = 0;
    }
}

File: Back/src/main/java/com/example/booking/project/models/entities/HotelModel.java
package com.example.booking.project.models.entities;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.hibernate.annotations.JoinColumnOrFormula;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
@Entity
@Table(name="Hotel")
public class HotelModel {

    @Id
    private Long hotel_id;

    @Column
    private String hotel_name;
    @Column
    private int max_reservations;
    @Column
    private String phone_number;
    @Column
    private String email;
    @Column
    private int capacity;
    @Column
    private String img_url;
    @Column
    private Long price;

}
